qutip/stochastic.py

```diff
@@ -280,28 +318,6 @@ class StochasticSolverOptions:
 
     progress_bar : :class:`qutip.ui.BaseProgressBar`
         Optional progress bar class instance.
-
-    *
-    time-dependent Qobj can be used for H, c_ops and sc_ops.
```

Comment:
I also can't find where this bit of docstring went, so checking that its removal was okay.

Reply:
I assumed that the `*` meant it was a footnote to a comment higher up, so I moved it up into the bumf at the top of the docstring, and mentioned it in the parameter descriptions of these guys.

Reply:
Looks good! Thanks for pointing out where it went to.